### DISCUSSION TITLE
Option to use default font antialiasing / smoothing

### DISCUSSION THREAD

**Discussion by tombull** [mantinedev/mantine#1359] [Feature requests]
Practically every time fonts are referenced in mantine, a method in the file `src/mantine-styles/src/theme/functions/fns/font-styles/font-styles.ts` is called that adds both the font family name and two directives that affect the way that the font is rendered: 
```
    WebkitFontSmoothing: 'antialiased',
    MozOsxFontSmoothing: 'grayscale',
```
This outputs the following CSS:
```
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
```
This changes the default type of antialiasing used in various browsers. In many cases this can make the font rendering subjectively better, however, in some cases it makes the font rendering worse instead of better.

Please can we have a theme option to disable this behaviour, for example on the theme object:
```
fontSmoothing: 'browserdefault' | 'custom';
```
(in this case, choosing `'browserdefault'` would omit the lines that are currently outputted, allowing use of the browser's default font rendering rather than the customised font rendering used when those lines are included).

Thoughts?

**Comment by rtivital** [comment]
in which cases font rendering becomes worse?

**Reply by tombull** [reply]
[This post](https://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/) gives a good summary of when it is suitable to use and when it is better not to use it.

**Reply by tombull** [reply]
There is also some good discussion on the topic [here](https://github.com/google/fonts/issues/1170)

**Reply by rtivital** [reply]
Okay, I'll change font smoothing styles to be defined on body element, you will be able to overwrite them with global styles.

**Reply by tombull** [reply]
That's a great solution. Thanks!
